src/web/app/components/student-list/student-list.component.scss

```diff
@@ -2,6 +2,15 @@
   text-align: center;
 }
 
+@media (max-width: 768px) { //applies only to mobile screens
+  .custom-button {
+    font-size: 14px;
+    width: 90px;
+    height: 30px;
```

Comment:
Don't think the height of the button was an issue before, and without this line the button height seems to increase... shall we remove it as well?